python/ray/tests/test_metrics.py

```diff
@@ -227,7 +227,7 @@ def getpid(self):
         if child_actor_info["state"] == -1:
             assert child_actor_info["requiredResources"]["CustomResource"] == 1
         else:
-            assert child_actor_info["state"] == 0
+            assert child_actor_info["state"] == 1
```

Comment:
> Why did this change? Was it wrong before?

That is because a new enum value 'ActorTableData::::ActorState::PENDING' is added. The value 0 is refers to  ActorTableData::::ActorState::PENDING while ActorTableData::::ActorState::ALIVE is expected in this unit test.

```
  enum ActorState {
    // Actor is pending.
    PENDING = 0;
    // Actor is alive.
    ALIVE = 1;
    // Actor is dead, now being reconstructed.
    // After reconstruction finishes, the state will become alive again.
    RECONSTRUCTING = 2;
    // Actor is already dead and won't be reconstructed.
    DEAD = 3;
  }
```